pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -916,6 +916,7 @@ public LocalBrokerData updateLocalBrokerData() {
             final SystemResourceUsage systemResourceUsage = LoadManagerShared.getSystemResourceUsage(brokerHostUsage);
             localData.update(systemResourceUsage, getBundleStats());
             updateLoadBalancingMetrics(systemResourceUsage);
+            updateLoadBalancingBundlesMetrics(getBundleStats());
```

Comment:
We'd better add a flag to control whether to expose bundle stats, because the bundle metrics will introduce a lot of metrics, which will increase the load of metric system.